packages/synchro-charts/src/components/charts/sc-webgl-base-chart/sc-webgl-base-chart.tsx

```diff
@@ -862,6 +867,7 @@ export class ScWebglBaseChart {
             hasNoDataPresent={hasNoDataPresent}
             hasNoDataStreamsPresent={hasNoDataStreamsPresent}
           />
+          {!hasSupportedDataTypes && <UnsupportedDataTypeStatus supportedDataTypes={this.supportedDataTypes} />}
```

Comment:
it looks from the logic that EmptyStatus can co-exist with UnsupportedDataTypeStatus - does that look weird?